bugbot/crash/analyzer.py

```diff
@@ -228,6 +231,7 @@ def regressed_by_potential_patches(self) -> Iterator[dict]:
             if changeset["max_score"] >= minimum_accepted_score
             and not changeset["is_merge"]
             and not changeset["is_backedout"]
+            and self._first_crash_date > parser.parse(changeset["push_date"])
```

Comment:
Could you add a comment mentioning that this is to reduce noise even if it could wrongly filter out some regressors (in the case a single signature refers to multiple crash causes).

Reply:
Done in a929257d756715933ec2672ac63da38ed66eeeaf.